docs/business-apps/power-automate/guidance/working-with-send-sp-http-request.md

```diff
@@ -4,22 +4,21 @@ ms.date: 03/11/2020
 ---
 
 # Working with the SharePoint Send HTTP Request flow action in Power Automate
-
 The SharePoint Send HTTP Request flow action in Power Automate lets you construct and execute SharePoint REST API queries. This action is particularly useful in cases where the existing SharePoint flow actions do not handle your requirements, or the action you are looking for is not yet available in the SharePoint connector.
 
+![Send an HTTP Request to SharePoint action](../../../images/flow-send-http-request-to-sp-action.png)
+
 See the following tips about how to work effectively with the SharePoint Send HTTP Request action.
```

Comment:
To work effectively with the SharePoint Send HTTP Request action, see the following tips.